phases/13-libffi.sh

```diff
@@ -47,7 +47,7 @@ rm -rf $TARGET
   --disable-docs \
   CC="$MSVCC" CXX="$MSVCC" LD=link \
   CPP="cl -nologo -EP" CXXCPP="cl -nologo -EP" \
-  CPPFLAGS="-DFFI_BUILDING_DLL" LDFLAGS="" \
+  CPPFLAGS="-DFFI_BUILDING_DLL" LDFLAGS="" CFLAGS="" \
```

Comment:
Nit: can we keep CPP + C flags together?
```suggestion
  CPPFLAGS="-DFFI_BUILDING_DLL" CFLAGS="" LDFLAGS="" \
```